documentation/src/main/wiki/UpgradeNotes.md

```diff
@@ -7,3 +7,29 @@ You may see all changes in the [Changelog](https://github.com/Cognifide/aet/blob
 
 ## Unreleased
 
+[PR-number](link) Pull request's name
+
+With the update osgi annotations to 6.0.0 version we had to change a little bit variable names. Currently, your config could has old names and you have to update them. Please follow instruction below:
+
+Open `aet\osgi-dependencies\configs\src\main\resources`
+
+In all files in this directory change variable names in way presented below:
+```
+MongoURI -> mongoURI
```

Comment:
Please specify which configurations should be updated.

Reply:
I changed it. Please check if i did it in a correct way